main.go

```diff
@@ -61,6 +69,34 @@ func RootCmd() *cobra.Command {
 	return rootCmd
 }
 
+func generateConfigs(tagsArg string, kinesisStreamNameArg string) FluentbitConfig {
+	tags, err := getTags(jsonConfigPath, tagsArg)
+	check(err, "tags not found")
+
+	kinesisStreamName, err := getKinesisStreamName(tags, kinesisStreamNameArg)
+	check(err, "kinesis stream name not found")
+
+	systemdUnit, systemdUnitLookupError := getSystemdUnit(tags)
+
+	if systemdUnitLookupError != nil {
+		fmt.Printf("%s: %v\n", "Application log shipping will not be configured", systemdUnitLookupError)
+		placeholders := map[string]string{"KINESIS_STREAM": kinesisStreamName, "APPLICATION_LOGS": ""}
+		config := replaceReplaceholders(fluentbitConfig, placeholders, tags)
+		return FluentbitConfig{
+			MainConfigFile: config,
+		}
+	} else {
```

Comment:
Nitpick, but in Go it's idiomatic to not wrap the second part in `else` in cases like this but just to rely on the early return. I.e.

```
if something {
  ...
  return x
}

...
return y
```

It's considered more readable and consistent with the general liking of early returns in Go. (Which is a strong contrast to Scala where early return isn't really a thing at all.)

Reply:
Thanks for the tip!